.github/workflows/sync-release-tags.yml

```diff
@@ -0,0 +1,30 @@
+on:
+  schedule:
+    # hourly
+    # This crontab is related to https://github.com/kevincobain2000/gobrew/issues/45
+    - cron: '0 * * * *'
+name: "Sync Go releases"
+
+jobs:
+  sync-tags:
+    name: sync
+    runs-on: ubuntu-latest
+    steps:
+      # curl to a tags to a fail
+      - name: curl latest tags
+        run: |
+         curl https://api.github.com/repos/golang/go/git/refs/tags > golang-tags.json
+
+      # use jq to validate the tags are correct and exit 1 upon failure
+      - name: Validate tags
+        uses: sergeysova/jq-action@v2
+        with:
+          cmd: cat golang-tags.json | jq -e --slurp '.[0][0].ref != null'
```

Comment:
Will exit 1, if curl failed, or the desired data was not found. Preventing unwanted updates to the json file.